DIPs/DIP1009.md

```diff
@@ -38,6 +38,21 @@ change.  An existing, well-known issue or a use case for an existing projects
 can greatly increase the chances of the DIP being understood and carefully
 evaluated.
 
+### Proposed New Syntax
+
+```
+ValueFunctionLiteral:
+    [ CaptureParameterList_opt ] ParameterMemberAttributes FunctionLiteralBody
+    [ CaptureParameterList_opt ] Identifier => Expression
+
+CaptureParameterList:
+    CaptureParameter
+    CaptureParameter , CaptureParameterList
+
```

Comment:
We should also have an AssignExpression here, i.e. for named captures (would be required for rvalues):
```d
[x = move(y)]() { return x.z }
```